src/main.cpp

```diff
@@ -32,7 +34,7 @@ static uint celltextcolors[] = {
 };
 #define CUSTOMCOLORIDX 0
 
-enum { TS_VERSION = 19, TS_TEXT = 0, TS_GRID, TS_BOTH, TS_NEITHER };
+enum { TS_VERSION = 20, TS_TEXT = 0, TS_GRID, TS_BOTH, TS_NEITHER };

```

Comment:
File structure gets changed so this will make it incompatible with the current version

Reply:
It's still backwards compatible though (old files will load, existing files will use PNG types for existing images), but not forwards compatible (old TS will not be able to load files with jpegs in them), correct?